lib/pipelines/compileSoy.js

```diff
@@ -108,17 +108,27 @@ function extractParams() {
 			var parsed = getParsedSoy(soyJsPath, file.contents);
 
 			templateData[soyJsPath] = {};
-			parsed.templates.forEach(function(cmd) {
-				if (cmd.deltemplate) {
+
+			parsed.body.forEach(function(template) {
+				if (template.deltemplate) {
```

Comment:
Makes sense, yeah, didn't think about it. Will update it and add a test for it, thanks!